packages/mdc-chips/README.md

```diff
@@ -211,12 +211,13 @@ To use the `MDCChip` and `MDCChipSet` classes, [import](../../docs/importing-js.
 
 Method Signature | Description
 --- | ---
-`get foundation() => MDCChipFoundation` | Returns the foundation
 `isSelected() => boolean` | Proxies to the foundation's `isSelected` method
 `beginExit() => void` | Begins the exit animation which leads to removal of the chip
 
 Property | Value Type | Description
 --- | --- | ---
+`foundation` | MDCChipFoundation | The foundation
+`shouldRemoveOnTrailingIconClick` | Boolean | Proxies to the foundation's `shouldRemoveOnTrailingIconClick`/`setShouldRemoveOnTrailingIconClick` methods
```

Comment:
Need to add `get` to the foundation method name in the description.

Also, an additional thought: add a sentence here explaining that if this is set to `false`, manually call `beginExit` on the chip component to control when to remove it.

Reply:
Sounds good, added as a `NOTE` to stick with having one line per table entry